render/gles2/pixel_format.c

```diff
@@ -39,10 +39,41 @@ static const struct wlr_gles2_pixel_format formats[] = {
 		.gl_type = GL_UNSIGNED_BYTE,
 		.has_alpha = true,
 	},
+		// 10 bit formats
+	{	
+		.wl_format = WL_SHM_FORMAT_ARGB2101010,
+		.depth = 32,
+		.bpp = 32,
+		.gl_format = GL_BGRA_EXT,
+		.gl_type = GL_UNSIGNED_BYTE,
```

Comment:
This can't work. I think it needs `GL_UNSIGNED_INT_2_10_10_10_REV`, which is only available in GLES3.

We need a way to test it to make sure it works anyways.